translations/de/hints/bryukh.html

```diff
@@ -1,10 +1,10 @@
 <body data-author="bryukh">
 <div class="question-answer">
     <div class="question">
-        I have no idea how to start solving this mission
+        Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
     </div>
     <div class="answer">
-        You can use a sequence of symbols or digraphs with values.
+        Du kannst eine Folge von Zeichen oder Digrafen mit Werten verwenden.
```

Comment:
For me the word _Digraf_ makes no sense here, same in english 😳"Verbindung von zwei Buchstaben zu einem Laut". Why not "Zeichen oder Symbole". Because at least description's english version uses _symboles_ once.